code/game/objects/structures/plasticflaps.dm

```diff
@@ -71,7 +71,8 @@
 
 /obj/structure/plasticflaps/CanAllowThrough(atom/movable/A, turf/T)
 	. = ..()
-
+	if(A.pass_flags & PASSFLAPS) //For anything specifically engineered to cross plastic flaps.
```

Comment:
what do mulebots use to pass through plastic flaps?

CAN mulebots even pass through plastic flaps in the first place?

Reply:
`		if(isbot(A)) //Bots understand the secrets
			return TRUE`